### PR TITLE
Some 1.13 fixes

### LINKED ISSUE #51: 1.13.2 Grape bushes
Grape bush sappling will grow into a leaf block however upon growing a grape, the bush (and sapling) are removed.  If I'm not mistaken, the grape should populate onto the leaf, and the leaf should then grow a new grape once the current grape is harvested.

### PR DESCRIPTION
**Pull request by SoSeDiK**
Final steps towards 1.13 update.
Evething seems to be working fine.
_______________________________________
Maybe fixes #51 where bushes dissappear duaring leaves decay. I didn't understand the issue completely.
Should prevent issue reports like #46 and #56.

### CODE REVIEW

File: src/me/mrCookieSlime/ExoticGarden/PlantsListener.java [hunk 1]
```diff
@@ -220,6 +222,15 @@ public void onHarvest(BlockBreakEvent e) {
 
 	@EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
 	public void onDecay(LeavesDecayEvent e) {
+		SlimefunItem sfItem = BlockStorage.check(e.getBlock());
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/ExoticGarden/PlantsListener.java]: This method gets the item's id, then looks up the item.
If you are only using this for id comparison, then you should rather only use the method that returns the ID.

**Comment by AeSix** [comment]
Is there plans or thoughts to change this from leaf block to berry bush for 1.14?
I don't know how that'd work with heads (possibly tag the berry and replace the item with the head?)

I believe the issue is due to changes in the way leaf decay works with 1.13, where a leaf requires a log in order to maintain existence.  This might require changes to the leaf block if not already.  

I will gladly test the new builds on my server.  My players have had a wonderful time with these plugins, even with the bugs and issues, and they'll be more than happy to see some mainline progress!  

Thank you so very much to both of you! <3

**Comment by SoSeDiK** [comment]
@AeSix, thanks!
There shouldn't be any issues with leaves anymore as I denied leaves decaying for bushes.
But I find weird that leaves didn't decay in 1.12.

**Comment by AeSix** [comment]
https://minecraft.gamepedia.com/18w22a#Changes

```Version exclusive Naturally-generated leaves now survive at a distance of up to 6 blocks from logs, instead of 4.
    The block state for leaves changed from a check_decay and decayable Booleans to distance (ranging from 1 to 7) and a persistent Boolean.```
This is probably the issue that you've overcome.  I'll have my players test with all of the bushes :)

**Comment by AeSix** [comment]
An update:
I've had several players test for the last several days.

The good
Bushes seem to be working properly, not losing their leaf block completely upon harvesting the fruits.
Some kitchen stuff that wasn't working is now working (chocolate, and some other recipes)

The bad
We've not been able to test everything.   I don't know if everything is working, but everything my players have tried does work.  Except Apple trees. 

The ugly
Oak Apple saplings turn into standard Minecraft oak trees.  No apple heads are populated, and the leaves drop MC oak saplings.  This I believe has been reported.  it's still an issue.

**Comment by SoSeDiK** [comment]
As for oak trees, try removing your schematics and let plugin generate the new ones.

**Comment by AeSix** [comment]
@SoSeDiK it appears that the leaf blocks are decaying again with EG dev build 8.

CS-CoreLib dev 36
Slimefun dev 89
SlimefunLuckyBlock dev 5 

(only things that have changed)

**Comment by AeSix** [comment]
Additionally, this is spamming the console:
```
[11:21:04 WARN]: [Slimefun] Task #1964622 for Slimefun vDEV - 89 (git 33073a41) generated an exception
java.lang.NullPointerException: null
```

**Pull request merged**